src/js/game/LevelMVC/LevelView.js

```diff
@@ -2247,6 +2248,12 @@ module.exports = class LevelView {
         break;
     }
 
+    // If we're making a charged redstoneWire, we need to do this again to make the sparkles
+    if (blockType.startsWith('redstoneWire') && blockType.endsWith('On')) {
+      // Please excuse the magic numbers. They're just to offset the sparkle correctly within the index.
+      sprite.addChild(this.createBlock(group, 0.35, 0.05, "redstoneSparkle"));
```

Comment:
I think this actually causes us to unnecessarily add the sprite to the plane group.  Here's what I was suggesting: https://github.com/code-dot-org/craft/pull/398 (I need to resolve conflicts for the most recent `randomInt` changes).